gio-sys/src/manual.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright 2013-2020, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use libc;
+use libc::c_int;
+
+pub type GSocketFamily = c_int;
+pub const G_SOCKET_FAMILY_INVALID: GSocketFamily = 0;
+pub const G_SOCKET_FAMILY_UNIX: GSocketFamily = 1;
+pub const G_SOCKET_FAMILY_IPV4: GSocketFamily = libc::AF_INET;
+pub const G_SOCKET_FAMILY_IPV6: GSocketFamily = libc::AF_INET6;
```

Comment:
Looking at the CI failures, seems like we need to invent values on Windows... not sure why `libc` doesn't have the values, the Windows libc (or more exact: winsock2) definitely has them. see e.g. https://docs.microsoft.com/en-us/windows-hardware/drivers/network/af-inet

Reply:
I wonder if there's a crate for winsock2 for this purpose

Reply:
Yes, [`winapi`](https://docs.rs/winapi/0.3.9/winapi/shared/ws2def/constant.AF_INET.html)

Reply:
Goes beyond that... I wrote that AF_INVALID and AF_UNIX didn't have a value in libc, but turns out they have a value on Linux, just not on MacOS. Kinda defeats the purpose of having those constants at all.

Reply:
I'm also checking with libc if a pr with the windows constants would be welcome.
I'd wait that answer and decide what is the cleanest approach, if there's no hurry on this.

Reply:
Sounds good. It's broken since a long time so waiting a bit longer is not going to make much of a difference :)

Can you link to any PRs/issues to libc or other crates related to this here? Also I think for Windows it makes sense to *not* have this as part of `libc` as it's in a separate library there for whatever reason.

Reply:
Sure, here is the "exploratory" issue: https://github.com/rust-lang/libc/issues/1890

Reply:
So based on that, let's go with `winapi` for Windows here. For macOS, is there also a problem?